### PR TITLE
Address oddities introduces by Assert.Multiple NUnit Analyzer Fixer

### PR DESCRIPTION
**Pull request by lailabougria**
Follow through for https://github.com/Particular/ServiceControl/pull/4377

### CODE REVIEW

File: src/Particular.LicensingComponent.UnitTests/AuditQuery_Tests.cs [hunk 1]
```diff
@@ -32,14 +35,15 @@ public async Task Should_return_known_endpoints_if_any()
 
         //Act
         var endpoints = await auditQuery.GetKnownEndpoints(default);
+        var serviceControlEndpoints = endpoints.ToList();
```
Comment (danielmarbach) [inline src/Particular.LicensingComponent.UnitTests/AuditQuery_Tests.cs:38]: I think this name is meaningless. I would much rather keep endpoints and change it to something like

        var endpoints = (await auditQuery.GetKnownEndpoints(default)).ToList();

File: src/Particular.LicensingComponent.UnitTests/ThroughputCollector/ThroughputCollector_Report_EnvironmentInformation_Tests.cs [hunk 2]
```diff
@@ -32,9 +32,10 @@ await DataStore.CreateBuilder()
         // Assert
         Assert.That(report, Is.Not.Null);
         Assert.That(report.ReportData.EnvironmentInformation, Is.Not.Null, $"Environment information missing from the report");
-        Assert.That(report.ReportData.EnvironmentInformation.EnvironmentData, Is.Not.Null, $"Environment data missing from the report");
         Assert.Multiple(() =>
         {
+            Assert.That(report.ReportData.EnvironmentInformation.EnvironmentData, Is.Not.Null, $"Environment data missing from the report");
+
```
Comment (danielmarbach) [inline src/Particular.LicensingComponent.UnitTests/ThroughputCollector/ThroughputCollector_Report_EnvironmentInformation_Tests.cs:38]: Given that the additional newline was removed in quite a few other places what is the benefit of having it here?

File: src/Particular.LicensingComponent.UnitTests/ThroughputCollector/ThroughputCollector_Report_EnvironmentInformation_Tests.cs [hunk 3]
```diff
@@ -63,9 +64,10 @@ await DataStore.CreateBuilder()
...
```
Comment (danielmarbach) [inline src/Particular.LicensingComponent.UnitTests/ThroughputCollector/ThroughputCollector_Report_EnvironmentInformation_Tests.cs:70]: Given that the additional newline was removed in quite a few other places what is the benefit of having it here?

File: src/Particular.LicensingComponent.UnitTests/ThroughputCollector/ThroughputCollector_Report_EnvironmentInformation_Tests.cs [hunk 4]
```diff
@@ -87,9 +89,10 @@ await DataStore.CreateBuilder()
...
```
Comment (danielmarbach) [inline src/Particular.LicensingComponent.UnitTests/ThroughputCollector/ThroughputCollector_Report_EnvironmentInformation_Tests.cs:95]: Given that the additional newline was removed in quite a few other places what is the benefit of having it here?

File: src/Particular.LicensingComponent.UnitTests/ThroughputCollector/ThroughputCollector_Report_EnvironmentInformation_Tests.cs [hunk 5]
```diff
@@ -118,9 +121,10 @@ await DataStore.CreateBuilder()
...
```
Comment (danielmarbach) [inline src/Particular.LicensingComponent.UnitTests/ThroughputCollector/ThroughputCollector_Report_EnvironmentInformation_Tests.cs:127]: Given that the additional newline was removed in quite a few other places what is the benefit of having it here?

File: src/Particular.LicensingComponent.UnitTests/ThroughputCollector/ThroughputCollector_Report_EnvironmentInformation_Tests.cs [hunk 6]
```diff
@@ -150,9 +154,10 @@ await DataStore.CreateBuilder()
...
```
Comment (danielmarbach) [inline src/Particular.LicensingComponent.UnitTests/ThroughputCollector/ThroughputCollector_Report_EnvironmentInformation_Tests.cs:160]: Given that the additional newline was removed in quite a few other places what is the benefit of having it here?

**Review by danielmarbach — DISMISSED** [review]
Only nits

File: src/ServiceControl.Audit.AcceptanceTests/Auditing/When_processed_message_is_imported.cs [hunk 7]
```diff
@@ -70,19 +70,18 @@ public async Task Should_be_accessible_via_the_rest_api()
...
                 Assert.That(auditedMessage.ProcessedAt, Is.Not.EqualTo(DateTime.MinValue), "Processed At should be correctly set");
 
-                Assert.That(TimeSpan.Zero, Is.LessThan(auditedMessage.ProcessingTime), "Processing time should be calculated");
+                Assert.That(auditedMessage.ProcessingTime, Is.LessThan(TimeSpan.Zero), "Processing time should be calculated");
+                Assert.That(auditedMessage.CriticalTime, Is.LessThan(TimeSpan.Zero), "Critical time should be calculated");
+
+                Assert.That(auditedMessage.MessageIntent, Is.EqualTo(MessageIntent.Send), "Message intent should be set");
             });
-            Assert.That(TimeSpan.Zero, Is.LessThan(auditedMessage.CriticalTime), "Critical time should be calculated");
-            Assert.That(auditedMessage.MessageIntent, Is.EqualTo(MessageIntent.Send), "Message intent should be set");
```
Comment (danielmarbach) [inline src/ServiceControl.Audit.AcceptanceTests/Auditing/When_processed_message_is_imported.cs:76]: I think these assertions were wrong in the first place and should be fixed. Expected and actual was mixed up and I think the condition should be that both times are GreaterThanZero

File: src/ServiceControl.Audit.Persistence.Tests.RavenDB/LicenseTest.cs [hunk 8]
```diff
@@ -37,7 +37,7 @@ public async Task EnsureLicenseIsValid()
                         Assert.That(details.Status, Is.EqualTo("Commercial"));
                         Assert.That(details.Expired, Is.False);
                         Assert.That(details.Type, Is.EqualTo("Professional"));
-                        Assert.That(DateTime.UtcNow.AddDays(14), Is.LessThan(details.Expiration), $"The RavenDB license expires {details.Expiration} which is less than 2 weeks. Contact RavenDB at <sales@ravendb.net> for the new license.");
+                        Assert.That(details.Expiration, Is.LessThan(DateTime.UtcNow.AddDays(14)), $"The RavenDB license expires {details.Expiration} which is less than 2 weeks. Contact RavenDB at <sales@ravendb.net> for the new license.");
```
Comment (danielmarbach) [inline src/ServiceControl.Audit.Persistence.Tests.RavenDB/LicenseTest.cs:40]: I think the way it was is correct. The idea is that the actual is two weeks from now and the expected is the expiration date to make the assertion fail when the license is about to expire. We flipped things now. If you want to flip expected and actual you need to adjust the check accordingly

**Review by cquirosj — APPROVED** [review]
I don't have comments other than the Raven license check that Daniel already pointed out.

**Comment by danielmarbach** [comment]
@lailabougria did you miss this one? https://github.com/Particular/ServiceControl/pull/4409#discussion_r1734963619

File: src/ServiceControl.Audit.AcceptanceTests/Auditing/When_processed_message_is_imported.cs (hunk 7 above)
Comment (lailabougria) [reply src/ServiceControl.Audit.AcceptanceTests/Auditing/When_processed_message_is_imported.cs]: Addressed now, sorry I missed this one

**Pull request merged**